### ISSUE TITLE
Accessibility problems with Mobile Center and Social Media page

### ISSUE THREAD

**Issue by cfarquharson** [wet-boew/GCWeb#494]
Ummmm, aren't you using tables for layout here: http://canada.ca/en/social/index.html  and here http://canada.ca/en/mobile/index.html There is no header row, so I suspect that the reason tables were use is just for the filter feature.

Also, on the social media page, all the links fail accessibility because they all say "Go to Social Media account" so that's a big fail there too.
